Private/Get-JenkinsUserInfo.ps1

```diff
@@ -0,0 +1,19 @@
+function Get-JenkinsUserInfo {
```

Comment:
Since the method is not called `current`

Then $Username should be used in the `"/user/$Username/api/json"` and if no username is given then use `me`?

Reply:
Otherwise, the method name is not deterministic in my opinion 😅

Reply:
Perhaps as part of `invoke-jenkins`, we should actually call this method and figure out who we are currently logged in as?

Reply:
This method looks up for a specific user under its `/user/me/api/json`. Then stores its userinfo under its username. 

The same goes to `Get-ADComputer`, where you can parse `-Identity` to specify which computer name to look up.